utils.py

```diff
@@ -73,11 +73,11 @@ def convert_hdf5_to_tif(paths, file_folders, data_folder):
         img_ds = {
             "marker": marker, 
             "nucleus_image": nucleus_image, 
-            "cells": cells, 
+            "cell_labels": cells, 
             "infected_labels": infected_labels,
             "serum_image": serum_image,
             "serum_image": serum_image, 
```

Comment:
Looks good to me. One question: Why do we have two `serum_image` objects here?

Reply:
removed duplicate line of code - had to make sure serum_image is definately present :D